frame/support/src/dispatch.rs

```diff
@@ -1837,6 +1831,16 @@ macro_rules! decl_module {
 			}
 		}
 
+		// Implement `OnGenesis` for `Module`
+		impl<$trait_instance: $trait_name $(<I>, $instance: $instantiable)?> $crate::traits::OnGenesis
+			for $mod_type<$trait_instance $(, $instance)?> where $( $other_where_bounds )*
```

Comment:
does this support multiple where bounds ?
```suggestion
			for $mod_type<$trait_instance $(, $instance)?> where $( $other_where_bounds ),*
```
But I see the same should be done for lots of impl above.

Reply:
No that is correct, we insert them correctly: https://github.com/paritytech/substrate/blob/bkchr-pallet-version-genesis/frame/support/src/dispatch.rs#L328

Reply:
ah yes